src/apps/dictionaries/components/DictionaryConceptsSummary.tsx

```diff
@@ -36,13 +55,40 @@ const DictionaryConceptsSummary: React.FC<Props> = ({ dictionary }) => {
   ).length;
   const customConceptCount = conceptReferences.length - fromPreferredSource;
 
+
+  const { updated_on = ''} = dictionary || {};
+  const updatedDate = getDate(updated_on);
```

Comment:
```suggestion
  const updatedDate = dictionary.updated_on;
```

Reply:
We also need to update the the `APIDictionary` type to have the `updated_on` property as a `string` property. One of the real advantages of Typescript, used properly, is that we can remove ambiguity around what type of data we're handling. In this case, `dictionary` is an instance of `APIDictionary` and it will have the `updated_on` property (that's guaranteed by the API) with a `string` value.

I'm also suggesting we don't reformat the string returned from the API. The reason is that the HEAD version of a dictionary can be modified multiple times on the same day, so we should just use the updated time stamp to determine the export version. It *might* be worth looking into what happens when a dictionary has just been created but never had any concepts added to it. In that instance `updated_on` might be `undefined` and we should fall back to `created_on`, which would mean adding *that* property to `APIDictionary` and changing this line to:

```ts
const updatedDate = dictionary.updated_on || dictionary.created_on;
```